File: Array/27. remove element/27.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        fast, slow = 0, 0
        for fast in range(len(nums)):
            if(nums[fast] != val):
                nums[slow] = nums[fast]
                slow += 1

        return slow

File: Array/27. remove element/27.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int slow = 0;
        int fast = 0;
        for(fast; fast < nums.size(); fast++){
            if(nums[fast]!=val){
                nums[slow] = nums[fast];
                slow++;
            }
        }
        return slow;
    }
};
